app/views/users/_form.html.erb

```diff
@@ -12,3 +12,8 @@
   <%= form.label :password %>
   <%= form.password_field :password, class: "input" %>
 </div>
+
+<div class="text-field">
+  <%= label_tag :sign_up_code %>
+  <%= text_field_tag :sign_up_code, nil, class: "input" %>
```

Comment:
Can we add some text to this form that tells people they can get their code by contributing on Patreon and linking to it?

Reply:
Good plan! Added this:

<img width="451" alt="Screenshot 2020-05-25 at 13 28 54" src="https://user-images.githubusercontent.com/1191840/82812844-c6224980-9e8b-11ea-8abe-4847879778c8.png">